code/modules/admin/fun_balloon.dm

```diff
@@ -49,18 +49,18 @@
 
 /obj/effect/fun_balloon/sentience/effect()
 	var/list/bodies = list()
-	for(var/mob/living/M in range(effect_range, get_turf(src)))
-		bodies += M
+	for(var/mob/living/possessable in range(effect_range, get_turf(src)))
+		if (!possessable.ckey && possessable.stat == CONSCIOUS) // Only assign ghosts to living, non-occupied mobs!
```

Comment:
what happens if they get given a ckey after this check but before the poll ends?

Reply:
Based on its current functionality I would guess it'll replace the ghosts in the mobs. I can't see this being especially common though.

Reply:
Doesn't mean it can't be easily prevented

Reply:
It's an easy fix, but this is a bug that's already existed, I personally think it's not necessary for this PR.